gradle.properties

```diff
@@ -2,3 +2,4 @@ version=5.3.0.BUILD-SNAPSHOT
 org.gradle.jvmargs=-Xmx1536m -Dfile.encoding=UTF-8
 org.gradle.caching=true
 org.gradle.parallel=true
+asciidoctorVersion=3.1.0
```

Comment:
No, I kinda agree with you about different versions in the future.

Also I don't like the change spreaded to several files which are fully not related to docs.
Can we, please, revert this commit?

Reply:
I didn't like it rolling into three files, either. I'll revert it.